public/wp-content/themes/glific/functions.php

```diff
@@ -99,9 +116,45 @@ function get_youtube_embed_url($url){
 	if (preg_match($shortUrlRegex, $url, $matches)) {
 		$youtube_id = $matches[count($matches) - 1];
 	}
-	return 'https://www.youtube.com/embed/' . $youtube_id. '?rel=0' ;
+	return 'https://www.youtube.com/embed/' . $youtube_id . '?rel=0';
+}
+
+function show_more_blogs()
+{
+	if (!isset($_POST['offset']) || empty($_POST['offset'])) {
+		wp_die();
```

Comment:
Why we are ending up the request without notifying the end-user?  is it a WP practice?

Reply:
@Satendra-SR this function is called from the ajax request. Once the error is sent out from here and received on the ajax call, the error message is sent out from there to the user. Let me know if you have any more queries on this.